packages/cfd/src/Containers/routes.jsx

```diff
@@ -2,7 +2,7 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { withRouter } from 'react-router';
 import BinaryRoutes from '../Components/Routes';
-import ErrorComponent from '../Components/Errors/error-component.jsx';
+import ErrorComponent from '../Components/Errors/error-component.tsx';
```

Comment:
Can we remove this extension part? Will it work?
```suggestion
import ErrorComponent from '../Components/Errors/error-component';
```

Reply:
it should still work